eve/worker/dockerfile/Dockerfile

```
@@ -30,6 +30,7 @@ RUN apt-get --assume-yes update \
 
 COPY requirements.txt /tmp/
 
+RUN pip3 install --upgrade pip
```

I would rather specify the version of pip you are installing, otherwise this image may not be stable in the future. I build it one day got one pip version you build it another day you get another, same story for the CI.

Change it to something like: 
```suggestion
ENV PIP_VERSION my.pip.version
RUN pip3 install pip==$PIP_VERSION
```